### ISSUE TITLE
Documentation for V1.3 boards

### ISSUE THREAD

**Issue by elbobius** [bigtreetech/BIGTREETECH-TMC2209-V1.2#18]
I could not find any manual/hardware documentation for the V1.3 driver boards. The boards layout differs from V1.2 and the soldering jumper used to enable UART/PDM seems to be gone with V1.3. Do the stock boards come with enabled uart? Adding some info for V1.3 would be great.

**Comment by hofingermarkus** [comment]
I think I found it here:

https://github.com/bigtreetech/BIGTREETECH-Stepper-Motor-Driver/tree/master/TMC2209

**Comment by Garywoo** [comment]
The manual does not label the VREF and INDEX pins, which is what I'm trying to find out. Does anyone have a confirmed pinout?

Pin 1: _I'm pretty sure this is VREF_
Pin 2: DIAG
Pin: 3: _I'm pretty sure this is INDEX_

<img width="653" height="660" alt="Image" src="https://github.com/user-attachments/assets/6b04021e-f6a1-450a-b84a-bbfbaebb254f" />

**Comment by Kletternaut** [comment]
https://global.bttwiki.com/img/TMC2209/TMC2209_Func.webp

![Image](https://github.com/user-attachments/assets/05d72321-8145-431b-99f5-ea59acf1514c)

https://global.bttwiki.com/TMC2209.html
https://biqu.equipment/products/bigtreetech-tmc2209-stepper-motor-driver-for-3d-printer-board-vs-tmc2208

**Comment by Garywoo** [comment]
@Kletternaut Thanks, I was able to find documented pinouts for V1.1 and V1.2 board versions, but not V1.3. I think the pinout of these 3 pins is the same between versions though, but would have liked to see it mentioned in documentation specifically related to the V1.3 boards.

**Comment by Kletternaut** [comment]
@Garywoo 
Yes, I can understand that, I felt the same way, which is why I wanted to confirm the pinout.
